.sequelizerc

```diff
@@ -0,0 +1,8 @@
+const path = require('path');
```

Comment:
The .sequelizerc file lets you specify the path to the migrations and models folder.